packages/pwa-buildpack/lib/cli/create-project.js

```diff
@@ -111,8 +100,7 @@ module.exports.builder = yargs =>
         .options({
             template: {
                 describe:
-                    'Name of a "template" to clone and customize. Currently only the "venia-concept" template is supported: `buildpack create-project --template venia-concept`',
-                choices: ['venia-concept']
+                    'Name of a "template" to clone and customize. Currently only the "venia-concept" template is supported: `buildpack create-project --template venia-concept`'
```

Comment:
Should we change this to the following? I don't know if `buildpack create-project --template venia-concept` would work anymore now that we resolve relative path or npm deployment.
```suggestion
                    'Name of a "template" to clone and customize. Currently only the "venia-concept" template is supported: `buildpack create-project --template "@magento/venia-concept"`'
```